### ISSUE TITLE
naming convention violations

### ISSUE THREAD

**Issue by danmarsden** [inspedralbes/moodle-block_attendancetable#3]
Classes and functions need to either be namespaced or must use the frankenstyle naming prefix 

eg:

```
namespace block_attendancetable/output
class course_info {
```

or 
```
class block_attendancetable_course_info {
```

In an Ideal world - you'd use the top example - you should also put classes into their own standalone files - eg one file with a course_info class, one file with a attendance_percentages class etc etc.

More info on this is here:
https://moodledev.io/general/development/policies/codingstyle/frankenstyle

Note that use of correct naming conventions is a blocker for plugins db approval if you're keen to get this into the plugins db at some point!

**Comment by a22ale** [comment]
Issue fixed on commit [499179d](https://github.com/inspedralbes/moodle-block_attendancetable/commit/499179dcea678848055b0e83566cbfb58e296376)

**Comment by a22ale** [comment]
Thank you for letting us know about this issue, we have changed the locallib classes to be similar to your first example and we have moved the classes to their own files inside classes/output as you mentioned.
